ydb/_topic_writer/topic_writer_asyncio.py

```diff
@@ -68,10 +66,10 @@ def __del__(self):
         self._loop.call_soon(self.close)
 
     async def close(self):
-        async with self._lock:
-            if self._closed:
-                return
-            self._closed = True
+        if self._closed:
+            return
+
+        self._closed = True
```

Comment:
I mark the object as closed before for prevent:
1. Double-close the object
2. Use object after start closing process - it will unusable right after call close()